rstb/rstb-op-polarimetric-tools-ui/src/main/resources/org/csa/rstb/polarimetric/docs/operators/CPRVIOp.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
+<html>
+<head>
+    <title>Help - Compact-pol Radar Vegetation Index</title>
+    <link rel="stylesheet" href="../style.css">
+</head>
+<body>
+<table class="header">
+    <tbody>
+    <tr class="header">
+        <td class="header">&nbsp; Compact-pol Radar Vegetation Index Generation</td>
+        <td class="header" align="right"><a href="../general/Overview.html"><img src="../images/snap_header.jpg"
+                                                                                 border="0"></a></td>
+    </tr>
+    </tbody>
+</table>
+
+
+<h3>Compact-pol Radar Vegetation Index-CpRVI</h3>
+<hr><br>
+<p align="right"><img src="images/MRSLABlogo.png"></p>
+<p align="right">Microwave Remote Sensing Lab (MRSLab)</p>
+<p align="right">Indian Institute of Technology Bombay, India</p>
+<p align="right">Contributors: Dr. Dipankar Mandal et al.</p>
+<p align="right">Tel: +91-22-2576-7677</p>
+<p align="right">Date: 25 Nov 2020</p>
+<p align="right">E-mail: mrscsre@gmail.com; dipankar.agrilengg@gmail.com</p>
+<p align="right">URL: http://www.mrslab.in
+</p>
+
+<!-- <h3>Processor Prerequsities</h3>
+<ul>
+<li>Python 3.5 or Python 3.6 to be installed (e.g. C:\Python35) and configure path to SNAP.</li>
```

Comment:
These operators wouldn't require Python

Reply:
It was in a commented block. Anyway, I removed all these block commented sections from html help files..